composer.json

```diff
@@ -19,9 +19,9 @@
 
     "require": {
         "php":                      ">=5.3.2",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/finder":           "~2.3|~3.0",
-        "symfony/validator":        "~2.3|~3.0",
+        "symfony/framework-bundle": "~2.3|~3.1",
```

Comment:
But we have a Symfony 3.1 version and can't use this lib. Pls could you accept these changes.